### ISSUE TITLE
Contrast themes – Update help-block styles

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#8817]
The help block has a few issues in WHCM. We want to make / consider the following changes:

- Text color should be `CanvasText` in all variations (#8791)
- Icon should be using the relevant color (#8816)
- We should try different border options to see whether we can do something less reliant on color, that works well regardless of the user’s theming.

Screenshot for reference:

![Image](https://user-images.githubusercontent.com/877585/177786050-527ef08d-f1ab-400d-b5e4-9bd624e49bbf.png)

Figma comments, starting here: https://www.figma.com/file/O81E48wieOZ7e6AivvksWn?node-id=6737:4090#222158795, and further below.

**Comment by thibaudcolas** [comment]
Discussed today – Anuja tried those changes, will share screenshots.
